doc/source/whatsnew/v0.25.1.rst

```diff
@@ -152,7 +152,7 @@ ExtensionArray
 
 Other
 ^^^^^
-
+- Bug in :meth:`Series.replace` and :meth:`DataFrame.replace` when replacing timezone-aware timestamps using a dict-like replacer (:issue:`27720`)
```

Comment:
does this close any issues?

Reply:
Not AFAICT.  Searching the tracker doesn't show anything obvious and the xfail comment points to a closed PR.